lib/sidekiq_unique_jobs/on_conflict.rb

```diff
@@ -15,17 +15,14 @@ module SidekiqUniqueJobs
   # @author Mikael Henriksson <mikael@zoolutions.se>
   #
   module OnConflict
-    STRATEGIES = {
-      log: OnConflict::Log,
-      raise: OnConflict::Raise,
-      reject: OnConflict::Reject,
-      replace: OnConflict::Replace,
-      reschedule: OnConflict::Reschedule,
-    }.freeze
+    # A convenience method for using the configured strategies
+    def self.strategies
+      SidekiqUniqueJobs.strategies
```

Comment:
I've used delegation a lot in many places but I honestly prefer a defined method and was planning on refactoring in this direction in some places 👍